graphql/schemas/Inventory/variant.graphql

```diff
@@ -159,6 +165,14 @@ extend type Mutation @guard {
         @field(
             resolver: "App\\GraphQL\\Inventory\\Mutations\\Variants\\Variants@addToWarehouse"
         )
+    updateVariantInWarehouse(
+        id: ID!
+        warehouse_id: Int!
+        input: VariantsWarehousesInput!
```

Comment:
@arfenis should we add the warehouse_id into VariantsWarehousesInput ? so we only pass 2 params?